fileio/fileio.go

```diff
@@ -128,3 +134,10 @@ func ReadFileToSingleLineString(filename string) string {
 	}
 	return catInput
 }
+
+// panicOnErr will call a blank panic if the input error != nil
+func panicOnErr(err error) {
```

Comment:
I have grown to think that having a "common" package is not a good style, but does it make sense to have a public version of this function somewhere?

Reply:
I am not sure. I also tend to dislike the common package, especially since 90% of common imports are just to use ExitIfError. I don't know what is idiomatic in this case and I don't have very strong feelings either way but I am slightly leaning towards each package that wants to use this kind of function implementing their own. That said I am happy to have my mind changed if there are reasons a public function would be better.